src/server/bg_services/bucket_logs_upload.js

```diff
@@ -0,0 +1,122 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const _ = require('lodash');
+const system_store = require('../system_services/system_store').get_instance();
+const dbg = require('../../util/debug_module')(__filename);
+const system_utils = require('../utils/system_utils');
+const config = require('../../../config');
+const P = require('../../util/promise');
+const cloud_utils = require('../../util/cloud_utils');
+const fs = require('fs');
+
+const BUCKET_LOGS_PATH = '/log/noobaa_bucket_logs/';
```

Comment:
What will happen if this path does not exist?

Reply:
I think it should just throw an exception and continue..